tests/dummy/app/controllers/chips.js

```diff
@@ -0,0 +1,57 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+  fruitNames: Ember.A(['Apple', 'Banana', 'Orange']),
+
+  customFruitNames: Ember.A(['Apple', 'Banana', 'Orange']),
+
+  contacts: Ember.A([{
+    name: 'James',
+    email: 'james@ember.com',
+    image: 'tomster.png'
+  }]),
+
+  allContacts: Ember.A([{
+    name: 'James',
+    email: 'james@ember.com',
+    image: 'tomster.png'
+  }, {
+    name: 'Chris',
+    email: 'chris@ember.com',
+    image: 'tomster.png'
+  }]),
+
+  remainingContacts: Ember.computed('allContacts.@each.email', 'contacts.@each.email', function() {
+    return this.get('allContacts').filter((source) => {
+      return !this.get('contacts').any(function(myContact) {
+        return source.email === myContact.email;
+      });
+    });
+  }),
+
+  actions: {
```

Comment:
It would be helpful to display in the demo what the result of the component's data is, I think.